test/kubeclient_test.rb

```diff
@@ -3,6 +3,7 @@
 require 'webmock/minitest'
 require './lib/kubeclient'
 
+TEST_FILE_PATH = File.dirname(__FILE__)
```

Comment:
I am invoking it with:

    rake test

but obviously I don't want to embed fixed paths such as:

    json_watch_stream = File.new('test/watch_stream.json')

I prefer to discover the path and use:

    json_watch_stream = File.new(File.join(TEST_FILE_PATH, 'watch_stream.json'))

so it's possible to invoke rake from any path and there's no assumption on where the code, Rakefile and tests are.